ui/widgets/nuxeo-card.js

```diff
@@ -53,7 +53,7 @@ import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
           :host {
             display: block;
             box-sizing: border-box;
-            background-color: white;
+            background-color: red;
```

Comment:
this is just a test, right?

Reply:
Yes, I just removed the commit with a force push 👍